maintainers/team-list.nix

```diff
@@ -48,6 +48,13 @@ with lib.maintainers; {
     scope = "Maintain Cinnamon desktop environment and applications made by the LinuxMint team.";
   };
 
+  chia = {
+    members = [
+      lourkeur
```

Comment:
@Atemu may I have your permission to include you in this team?

Reply:
Sure!

Reply:
Done!